app/components/QuestionCard/index.js

```diff
@@ -5,40 +5,16 @@
  */
 
 import { createStyleSheet } from 'jss-theme-reactor';
-import Avatar from 'material-ui/Avatar';
+
 import Card, { CardContent } from 'material-ui/Card';
 import Divider from 'material-ui/Divider';
-import List, { ListItem } from 'material-ui/List';
-import { grey } from 'material-ui/styles/colors';
-import Typography from 'material-ui/Typography';
+import List from 'material-ui/List';
 import customPropTypes from 'material-ui/utils/customPropTypes';
 import moment from 'moment';
 import React, { PropTypes } from 'react';
-import Markdown from '../Markdown';
-
-const styleSheet = createStyleSheet('SimpleCard', {
-  avatar: {
-    marginRight: 16,
-    flexShrink: 0,
-  },
-  avatarH: {
-    extend: 'avatar',
-    marginTop: 8,
-    transform: 'scale(1.5)',
-  },
-  lItem: {
-    alignItems: 'flex-start',
-    paddingTop: 12,
-    paddingRight: 16,
-    paddingBottom: 12,
-    paddingLeft: 16,
-    backgroundColor: grey[50],
+import QuestionRow from '../QuestionRow';
```

Comment:
przeniósł bym to jako subcomponent QuestionCarda co by nie mnożyć top level elementów (patrz ContestTable)